app/Cart.php

```diff
@@ -70,10 +70,10 @@ public function removeItem($item)
     public function updateQuantity($item, $qty)
     {
         // Count the number of $items in array
-        $filteredItems = array_filter($this->items, function($value) use ($item) { return $value === $item; });
```

Comment:
What does `array_filter` do?

List the steps you think we need to take to get this to pass.